src/st/st-entry.c

```diff
@@ -241,10 +245,10 @@ st_entry_dispose (GObject *object)
   StEntryPrivate *priv = ST_ENTRY_PRIV (entry);
   GdkKeymap *keymap;
 
-  if (priv->entry)
```

Comment:
Sorry to nitpick, but I think this change should be separated out into another commit. It's not related to this IIRC -- see https://github.com/endlessm/eos-desktop/commit/b27e85340f197d82da5372a1f26baaff27e3f4d1